### DISCUSSION TITLE
Workflow improvement for multiple projects

### DISCUSSION THREAD

**Discussion by arky** [digidem/mapeo-mobile#778] [Ideas]
It would be a good idea to able to organize by projects on Mapeo-mobile. For instance, a health researcher could be involved with  multiple projects, they often use the same device to collect the information. For instance, a health worker could be collecting information HIV-Health project and also Malaria project. 

If the person could quickly switch projects (filter by project) then it would be quite beneficial. At the moment, it  can become quite confusing if you are trying to work on multiple projects using a single device.

**Comment by gmaclennan** [comment]
This is high up on our priorities list for Mapeo. We have come across the same challenges of managing data from multiple projects. Project support is also part of making initial setup easier, particularly around setting up shared secret keys.
